scripts/madrid/predictions_collapsed/Snakefile

```diff
@@ -0,0 +1,134 @@
+'''
+Produce a collapsed feature representation on Madrid Transfer to ICU Prediction
+
+
+'''
+
+# Default environment variables
+# Can override with local env variables
+
+MADRID_VERSION = os.environ.get('MADRID_VERSION', 'v20200424')
+PROJECT_REPO_DIR = os.environ.get("PROJECT_REPO_DIR", os.path.abspath("../../../"))
+PROJECT_CONDA_ENV_YAML = os.path.join(PROJECT_REPO_DIR, "ts_pred.yml")
+
+MADRID_DATASET_TOP_PATH = os.path.expandvars(os.path.join("$HOME", "datasets/"))
+SITE_NAME = "HUF_maxN21000/"
+MADRID_DATASET_STD_PATH = os.path.join(MADRID_DATASET_TOP_PATH, MADRID_VERSION, SITE_NAME)
+
+rule all:
+    input:
+        os.path.join(MADRID_DATASET_STD_PATH, 'alignedfeatures_per_tstep.csv'),
+        os.path.join(MADRID_DATASET_STD_PATH, 'normfeatures_per_tstep.csv'),
+        os.path.join(MADRID_DATASET_STD_PATH, 'CollapsedFeaturesPerSequence.csv'),
+        os.path.join(MADRID_DATASET_STD_PATH, 'Spec_CollapsedFeaturesPerSequence.json'),	
+        os.path.join(MADRID_DATASET_STD_PATH, 'x_train.csv'),
+        os.path.join(MADRID_DATASET_STD_PATH, 'x_test.csv'),
+        os.path.join(MADRID_DATASET_STD_PATH, 'y_train.csv'),
+        os.path.join(MADRID_DATASET_STD_PATH, 'y_test.csv')
+
+rule align_to_grid:
```

Comment:
@rathp Not sure we need to do "align to grid". Can do collapsing directly on original features

Reply:
Removed align_to_grid from feature collapsing workflow. Collapsing is now done using hours_from_admission as timestamps without any aligning to grid.